### PR TITLE
Added Project

### PR DESCRIPTION
**Pull request by neeraj-patidar**

### CODE REVIEW

File: entry/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "entry",
```
Comment (samuel-abhishek) [inline entry/package-lock.json:2]: Delete the generated files.

File: entry/src/main/ets/MainAbility/pages/filled_sample.ets [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        placeholderColor:Color.Red,
+        caretColor:Color.Red,
+         },
+      trailingIconClick: (event) => {
+        AlertDialog.show({ message: 'trail icon click\n' + JSON.stringify(event.target) })
+      },
+      leadingIconClick: (event) => {
+        AlertDialog.show({ message: 'lead icon click\n' + JSON.stringify(event.target) })
+      },
+//      onChanged: () => {
```
Comment (samuel-abhishek) [inline entry/src/main/ets/MainAbility/pages/filled_sample.ets:45]: Remove the commented code. If this code is required, then add a different use case within the sample app. (Applies to all commented code in the sample app)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+## Material_UI_TextField
+### Features
```
Comment (samuel-abhishek) [inline README.md:2]: Add a gif showing the functionality of both outlined and filled material text fields.

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please check and handle the review comments

File: MaterialTextField/package.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "MaterialTextField",
```
Comment (archanaatr) [inline MaterialTextField/package.json:2]: try to change to material-textfield

File: entry/package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  "ohos": {
+    "org": "huawei",
+    "buildTool": "hvigor",
+    "directoryLevel": "module"
+  },
+  "description": "example description",
+  "repository": {},
+  "license": "ISC",
+  "dependencies": {
+    "@ohos/MaterialTextField": "file:../MaterialTextField"
```
Comment (archanaatr) [inline entry/package.json:13]: define as @ohos/materialtextfield

**Review by archanaatr — COMMENTED** [review]
pls chk the comments.

**Review by archanaatr — APPROVED** [review]
ok

**Pull request merged**